core/font.lua

```diff
@@ -40,11 +41,12 @@
         SILE.process(content)
         SILE.settings.popState()
       end
-    end, "Set current font family, size, weight, style, variant, script, direction and language")
+    end, "Set current font family, size, weight, width, style, variant, script, direction and language")
 
 SILE.settings.declare({name = "font.family", type = "string", default = "Gentium"})
 SILE.settings.declare({name = "font.size", type = "number or integer", default = 10})
 SILE.settings.declare({name = "font.weight", type = "integer", default = 200})
+SILE.settings.declare({name = "font.width", type = "number or integer", default = 100})
```

Comment:
I would second @behdad's request for this to be a 1.0 based scale factor rather than a percentage.

Reply:
Based on current discussion, this should be changed because the type is not right (could be a percent or a keyword). We will need a new type and matching cast/validate functions to handle this—unless it turns out we can just pass raw values through and get sensible errors back from lower lever handling.